ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/TopicsDeserializer.java

```diff
@@ -37,23 +37,31 @@ public TopicsDeserializer(final Class<?> vc) {
   }
 
   @Override
-  public List<List<LogTopic>> deserialize(
+  public List<List<Bytes32>> deserialize(
       final JsonParser jsonparser, final DeserializationContext context) throws IOException {
     final JsonNode topicsNode = jsonparser.getCodec().readTree(jsonparser);
-    final List<List<LogTopic>> topics = Lists.newArrayList();
+    final List<List<Bytes32>> topics = Lists.newArrayList();
 
     if (!topicsNode.isArray()) {
-      topics.add(singletonList(LogTopic.fromHexString(topicsNode.textValue())));
+      topics.add(singletonList(Bytes32.fromHexStringLenient(topicsNode.textValue())));
     } else {
       for (JsonNode child : topicsNode) {
         if (child.isArray()) {
-          final List<LogTopic> childItems = Lists.newArrayList();
+          final List<Bytes32> childItems = Lists.newArrayList();
           for (JsonNode subChild : child) {
-            childItems.add(LogTopic.fromHexString(subChild.textValue()));
+            if (subChild.isNull()) {
```

Comment:
I was about to complain about the sudden handling of null, then I saw that this ugly logic used to be wrapped inside `LogTopic` and what was changing was just where it was done.  The struggle against nulls never ends and will need to be carried on in another PR.  No action needed, just a soliloquy.